crates/topos/src/tracing.rs

```diff
@@ -50,18 +39,14 @@ fn build_resources(otlp_service_name: String) -> Vec<KeyValue> {
     resources
 }
 
-fn create_filter(verbose: u8) -> EnvFilter {
-    if verbose > 0 {
-        EnvFilter::try_new(format!("warn,topos={}", verbose_to_level(verbose).as_str())).unwrap()
-    } else {
-        EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("warn,topos=info"))
-    }
+#[allow(dead_code)]
+fn create_filter() -> EnvFilter {
+    EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("warn,topos=info"))
 }
 
 // Setup tracing
 // If otlp agent and otlp service name are provided, opentelemetry collection will be used
 pub(crate) fn setup_tracing(
-    verbose: u8,
```

Comment:
why did you removed the `verbose` ?

Reply:
I didn't see it being used in any commands we used or people use! 

We can keep it but I wanted to clear up things we don't need!

Reply:
There is a `verbose` attribute on `NodeCommand` and it was present also on `TceCommand`.

This is a global attribute that are present for subcommand. It allows us to have logs when executing the command by passing a flag instead of an environment variable.

Reply:
I think a `verbose` argument is not so great, it's too coarse. What we really would want is a `--log` switch that takes the same syntax as `RUST_LOG`, allowing all of these to do the expected thing:

- `--log trace`: set `TRACE` level logging
- `--log tce=warn,sequencer=trace`: set `WARN` for the tce and `TRACE` for sequencer and leave `INFO` for all other components

Reply:
The fact is that having this approach bound the exposed log syntax of the inner architecture of the repo.

I don't have strong opinions on that but we need to be careful about how we design that, I find the `RUST_LOG` syntax powerful but annoying at the same time.

Reply:
Fair points. I reverted to the previous state when it comes to `verbose`, and we can decide in another PR on how we want to adjust/proceed with that?